mucrop.c

```diff
@@ -303,10 +303,13 @@ int main(int argc, const char *argv[])
 	struct timespec tp;
 	int ret = 0;
 
-	if (argc != 3) {
+	if (argc != 2 && argc != 3) {
 		usage(true);
 		return EX_USAGE;
 	}
+	if (argc == 2) {
+		dst_filename = src_filename;
+	}
```

Comment:
I'd probably prefer  using a switch here over (just looks prettier to me)
```suggestion
	switch (argc) {
		case 2:
			dst_filename = src_filename;
		case 3:
			break;
		default:
			usage(true);
			return EX_USAGE;
	}
```